Description: lxc-wait: initialize timeout to -1
 Otherwise it defaults to 0, meaning don't wait.  -1 means wait forever,
 which is what we want as the default behavior.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: yes

Index: lxc/src/lxc/lxc_wait.c
===================================================================
--- lxc.orig/src/lxc/lxc_wait.c	2012-08-30 11:30:48.999463000 -0500
+++ lxc/src/lxc/lxc_wait.c	2012-08-30 11:31:51.491554965 -0500
@@ -75,6 +75,7 @@
 	.options  = my_longopts,
 	.parser   = my_parser,
 	.checker  = my_checker,
+	.timeout = -1,
 };
 
 static void timeout_handler(int signal)
